backend/dal/Migrations/v01.06.01/Down/PreDown/01-NotificationTemplates.sql

```diff
@@ -0,0 +1,728 @@
+UPDATE dbo.[NotificationTemplates]
+SET [Body] = '<html><head><title>@Model.Environment.Title</title></head><body><p>Dear @Model.ToAgency.AddressTo,</p><p>Your project @Model.Project.ProjectNumber has been approved. Signin to <a href="@Model.Environment.Uri">PIMS</a> to review the progress.</p><p>Sincerely Real Property Division</p></body></html>'
+WHERE [Id] = 4
+
+UPDATE dbo.[NotificationTemplates]
+SET [Body] = '
+@using System.Linq
```

Comment:
so is this an html file in the database?

Reply:
Technically it's a Razor file in the database.

Reply:
Hi Devin, just to let you know that I just had a PR which is already using the v01.06.01 number.